zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -712,6 +712,49 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+  /process-instances/{processInstanceKey}/cancellation:
+    post:
+      tags:
+        - Process Instance
+      summary: Cancel process instance
+      description: Cancel a running process instance.
+      parameters:
+        - name: processInstanceKey
+          in: path
+          required: true
+          description: The key of the process instance that should be modified.
```

```suggestion
          description: The key of the process instance to cancel.
```
:question: Should this be cancel instead of modified?